File: web3rpc/rpc-specs/code-samples/javascript/governance/itemsAt.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumber = 89;

  provider.governance
    .itemsAt(blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/governance/itemsAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockTag = 0

w3 = Web3(Web3.HTTPProvider(host))
governance_response = w3.governance.items_at(blockTag)

print(governance_response)
